go/processor/floria/run_context.go

```diff
@@ -73,14 +95,36 @@ func (r runContext) Call(kind tosca.CallKind, parameters tosca.CallParameters) (
 		r.RestoreSnapshot(snapshot)
 	}
 
+	if kind == tosca.Create || kind == tosca.Create2 {
+		r.SetCode(createdAddress, tosca.Code(result.Output))
```

Comment:
This should probably be skipped (or reverted by the snapshot above) if the execution produced an error, right?